src/pages/development.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Development
+---
+
+You want to improve MapProxy, found a bug and want to fix it? Great! This document points you to some helpful information.
+
+#### Source
+
+Releases are available from the [PyPI project page of MapProxy](http://pypi.python.org/pypi/MapProxy). There is also [an archive of all official releases](http://pypi.python.org/packages/source/M/MapProxy/), unofficial releases and pre-releases [can be found here](https://mapproxy.org/static/rel/).
```

Comment:
Last two links are broken or outdated, i think we should remove them. Maybe this is a good place to also mention the docker images